src/Microsoft.Azure.SignalR/HubHost/ServiceHubConnectionContext.cs

```diff
@@ -30,10 +30,19 @@ internal class ServiceHubConnectionContext : HubConnectionContext
         public const string ItemsUnavailableError = nameof(Items) + UnavailableErrorTemplate;
         public const string ProtocolUnavailableError = nameof(Protocol) + UnavailableErrorTemplate;
 
+#if NETCOREAPP3_0
+        public ServiceHubConnectionContext(HttpContext context)
```

Comment:
cc @BrennanConroy @mikaelm12 did we break this?

Reply:
Yes
https://github.com/aspnet/AspNetCore/issues/11431

Reply:
Ah yes.